ts/Extensions/Annotations/NavigationBindings.ts

```diff
@@ -195,17 +195,24 @@ const bindingsUtils = {
         var chart = annotation.chart,
             options = annotation.options.typeOptions,
             coords = chart.pointer.getCoordinates(event),
-            width = coords.xAxis[0].value - options.point.x,
-            height = options.point.y - coords.yAxis[0].value;
-
-        annotation.update({
-            typeOptions: {
-                background: {
-                    width: chart.inverted ? height : width,
-                    height: chart.inverted ? width : height
+            coordsX = NavigationBindings.prototype.getAssignedAxis(coords, 'x'),
+            coordsY = NavigationBindings.prototype.getAssignedAxis(coords, 'y'),
```

Comment:
Any particular reason to use `NavigationBindings.prototype.getAssignedAxis` over `chart.navigationBindings.getAssignedAxis` in this one annotation?

I'm fine with static methods, but then assign it to `NavigationBindings` instead of `NavigationBindings.prototype`.